benchmarking/hwmcc17.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+# ca. 30 minutes
```

Comment:
What about using GNU parallel to run 4 jobs concurrently? Would that bring down execution time to almost a fourth? If so, I'd feel ok to have this in CI. Else, perhaps use a cron job to run it on a daily basis?

Reply:
It should parallelise well; I'll try.